index.js

```diff
@@ -1,12 +1,11 @@
 module.exports = {
-  plugins: ['html', 'json'],
   extends: [
-    '@fs/eslint-config-frontier-react/es6',
-    '@fs/eslint-config-frontier-react/react',
-    '@fs/eslint-config-frontier-react/jest',
-    '@fs/eslint-config-frontier-react/test-helpers',
-    // '@fs/eslint-config-frontier-react/noFixRules',
-    '@fs/eslint-config-frontier-react/dont-need-lodash',
-    '@fs/eslint-config-frontier-react/prettierSetup',
+    './react',
```

Comment:
I have a faint memory that I used to do it this way, but that it messes things up if I tried to do npm linking (for testing purposes) in zion/react apps. 

I could be wrong though.

Reply:
maybe. linking never seems to work right with me. I tested by installing from the github branch (npm i --sav-dev github:fs-webdev/eslint-config-frontier-react#no-codeclimate-dep) than from artifactory